src/D-FSL-1.0.xml

```diff
@@ -0,0 +1,380 @@
+          <b>(1)</b>
+          <p>Für entgegenstehende Rechte Dritter haften die Rechtsinhaber nur, sofern sie Kenntnis von diesen
+             Rechten hatten, ohne Sie zu informieren.</p>
+        </li>
+        <li>
+          <b>(2)</b>
+          <p>Die Haftung für Fehler und sonstige Mängel des Programms richtet sich nach den außerhalb dieser
+             Lizenz getroffenen Vereinbarungen zwischen Ihnen und den Rechtsinhabern oder, wenn eine solche
+             Vereinbarung nicht existiert, nach den gesetzlichen Regelungen.</p>
+          <p>§ 8 Verträge mit Dritten</p>
+        </li>
+        <li>
+          <b>(1)</b>
+          <p>Diese Lizenz regelt nur die Beziehung zwischen Ihnen und den Rechtsinhabern. Sie ist nicht
+             Bestandteil der Verträge zwischen Ihnen und Dritten.</p>
+        </li>
+        <li>
+          <b>(2)</b>
+          <p>Die Lizenz beschränkt Sie nicht in der Freiheit, mit Dritten, die von Ihnen Kopien des Programms
+             erhalten oder Leistungen in Anspruch nehmen, die im Zusammenhang mit dem Programm stehen,
+             Verträge beliebigen Inhalts zu schließen, sofern Sie dabei Ihren Verpflichtungen aus dieser
+             Lizenz nachkommen und die Rechte der Dritten aus dieser Lizenz nicht beeinträchtigt werden.
+             Insbesondere dürfen Sie für die Überlassung des Programms oder sonstige Leistungen ein Entgelt
+             verlangen.</p>
+        </li>
+        <li>
+          <b>(3)</b>
+          <p>Diese Lizenz verpflichtet Sie nicht, das Programm an Dritte weiterzugeben. Es steht Ihnen frei zu
+             entscheiden, wem Sie das Programm zugänglich machen. Sie dürfen aber die weitere Nutzung durch
+             Dritte nicht durch den Einsatz technischer Schutzmaßnahmen, insbesondere durch den Einsatz von
+             Kopierschutzvorrichtungen jeglicher Art, verhindern oder erschweren. Eine passwortgeschützte
+             Zugangsbeschränkung oder die Nutzung in einem Intranet wird nicht als technische
+             Schutzmaßnahme angesehen.</p>
+          <p>§ 9 Text der Lizenz</p>
+        </li>
+        <li>
+          <b>(1)</b>
+          <p>Diese Lizenz ist in deutscher und englischer Sprache abgefasst. Beide Fassungen sind gleich
+             verbindlich. Es wird unterstellt, dass die in der Lizenz verwandten Begriffe in beiden
+             Fassungen dieselbe Bedeutung haben. Ergeben sich dennoch Unterschiede, so ist die Bedeutung
+             maßgeblich, welche die Fassungen unter Berücksichtigung des Ziels und Zwecks der Lizenz am
+             besten miteinander in Einklang bringt.</p>
+        </li>
+        <li>
+          <b>(2)</b>
+          <p>Der Lizenzrat der Deutschen Freien Software Lizenz kann mit verbindlicher Wirkung neue Versionen
+             der Lizenz in Kraft setzen, soweit dies erforderlich und zumutbar ist. Neue Versionen der
+             Lizenz werden auf der Internetseite http://www.d-fsl.de mit einer eindeutigen Versionsnummer
```

Comment:
Upstream is inconsistent here, using [no wrapping chars in the HTML][1] and [`<>` in the text][2].   I recommend using `<alt>` or `<optional>` to cover both cases, and have no opinion on which form should be canonical.

[1]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/deutsche-freie-software-lizenz
[2]: https://www.hbz-nrw.de/produkte/open-access/lizenzen/dfsl/D-FSL-1_0_de.txt/view